<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutEpargneInterfaceClass</class>
 <widget class="QDialog" name="AjoutEpargneInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajoutd'un compte Épargne</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelNoCompte">
      <property name="text">
       <string>Numéro de Compte :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTauxInteret">
      <property name="text">
       <string>Taux d'intérêt :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelSolde">
      <property name="text">
       <string>Solde :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditNoCompte">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTauxInteret"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSolde"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDescription"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
